acquia_lift.admin.inc

```diff
@@ -0,0 +1,410 @@
+    '#required' => TRUE,
+  );
+  $form['acquia_lift_decision_api_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Decision API URL'),
+    '#description' => t('Your Lift Decision API\'s URL.  Unless explicitly instructed, leave empty to use default URL.'),
+    '#field_prefix' => 'https://',
+    '#default_value' => _acquia_lift_admin_clean_url(variable_get('acquia_lift_decision_api_url', '')),
+  );
+  $form['acquia_lift_oauth_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Authentication URL'),
+    '#description' => t('Your Lift Authentication API\'s URL.  Unless explicitly instructed, leave empty to use default URL.'),
+    '#field_prefix' => 'https://',
+    '#field_suffix' => '/authorize',
+    '#default_value' => _acquia_lift_admin_clean_url(_acquia_lift_admin_remove_authorize_suffix(variable_get('acquia_lift_oauth_url', ''))),
+  );
+  return $form;
+}
+
+
+/**
+ * Admin form for configuring acquia_lift behavior.
+ */
+function _acquia_lift_admin_build_identity_form(&$form, &$form_state) {
+  $identity_parameter_display_value = variable_get('acquia_lift_identity_param', 'identity');
+  $identity_type_parameter_display_value = variable_get('acquia_lift_identity_type_param', 'identityType');
+  $default_identity_type_display_value = variable_get('acquia_lift_default_identity_type', ACQUIA_LIFT_DEFAULT_IDENTITY_TYPE_IDENTIFIER);
+  $default_identity_type_default_value = variable_get('acquia_lift_default_identity_type', ACQUIA_LIFT_DEFAULT_IDENTITY_TYPE_DEFAULT);
+
+  $form['acquia_lift_identity_param'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Identity Parameter'),
+    '#description' => t('The query string parameter for specific visitor information, such as an email address or social media username, which is sent to the Acquia Lift service. Example: ?<ins>@identity_parameter_display_value</ins>=jdoe01', [
+      '@identity_parameter_display_value' => $identity_parameter_display_value,
+    ]),
+    '#default_value' => variable_get('acquia_lift_identity_param'),
+  );
+  $form['acquia_lift_identity_type_param'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Identity Type Parameter'),
+    '#description' => t('The query string parameter for the category (standard or custom) of the visitor\'s identity information. Example: ?@identity_parameter_display_value=jdoe01&<ins>@identity_type_parameter_display_value</ins>=@default_identity_type_default_value', [
+      '@identity_parameter_display_value' => $identity_parameter_display_value,
+      '@identity_type_parameter_display_value' => $identity_type_parameter_display_value,
+      '@default_identity_type_default_value' => $default_identity_type_default_value,
+    ]),
+    '#default_value' => variable_get('acquia_lift_identity_type_param'),
+    '#states' => array(
+      'visible' => array(
+        ':input[name="acquia_lift_identity_param"]' => ['!value' => ''],
+      ),
+    ),
```

Comment:
As per your recommendation, I have updated the D8 module to use the same '#states' for acquia_lift_identity_param. I think we can do that in here too.